core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -436,13 +438,20 @@ object GameStarter {
         var preferredTiles = freeTiles.toList()
         for (startBias in civ.nation.startBias) {
             preferredTiles = when {
-                startBias.startsWith("Avoid [") -> {
-                    val tileToAvoid = startBias.removePrefix("Avoid [").removeSuffix("]")
-                    preferredTiles.filter { !it.matchesTerrainFilter(tileToAvoid) }
+                startBias.equalsPlaceholderText("Avoid []") -> {
+                    val tileToAvoid = startBias.getPlaceholderParameters()[0]
+                    preferredTiles.filter { tile ->
+                        !tile.getTilesInDistance(1).any {
+                            it.matchesTerrainFilter(tileToAvoid)
+                        }
+                    }
                 }
-                startBias == Constants.coast -> preferredTiles.filter { it.isCoastalTile() }
                 startBias in tileMap.naturalWonders -> preferredTiles  // passthrough: already failed
```

Comment:
(not specific to this PR, but this just caught my eye)
Doesn't `tileMap.naturalWonders` only contain the wonders that are actually placed on this map? If so, aren't those the exact same that would have been filtered before this `for`-loop, effectively making this dead code? I'd assume you want to check if they are a natural wonder, and if so, not do anything, but that would be `ruleSet.terrains[startBias]!!.type == TerrainType.NaturalWonder`, right?

Additionally, the code above this PR for determining the starting space next to natural wonders has a variable `wonderNeighbor` which would make it likely to contain the tiles adjacent to natural wonders. However, unless I read the code incorrectly, it only finds the exact tile the wonder is on